19.3/basic/app.py

```diff
@@ -0,0 +1,72 @@
+from flask import Flask, request, render_template, redirect, flash, session
+from flask_debugtoolbar import DebugToolbarExtension
+from surveys import satisfaction_survey as survey
+
+RESPONSES_KEY = "responses"
+
+app = Flask(__name__)
+app.config['SECRET_KEY'] = "never-tell!"
+app.config['DEBUG_TB_INTERCEPT_REDIRECTS'] = False
+
+debug = DebugToolbarExtension(app)
+
+@app.route("/")
+def start_page():
+    """Home page to select a survey"""
+    
+    return render_template("start-page.html", survey=survey)
+
+@app.route("/begin", methods=["POST"])
```

Comment:
@Danielewin8 see how they set the POST method here, you could use that with Boggle exercise